Documentation/security/index.rst

```diff
@@ -24,3 +24,4 @@ Network Policy Security Tutorials
    aws
    policy-creation
    host-firewall
+   restrict-pod-access
```

Comment:
This is currently in the 1st commit, but the page is only added in the 4th commit.